CloudFlareDDNS.csproj

```diff
@@ -9,10 +9,11 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>CloudFlareDDNS</RootNamespace>
     <AssemblyName>CloudFlareDDNS</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>

```

Comment:
I'd like to keep this on 4.0 if possible. Is there anything in here requiring 4.7?
XP last supported by 4.03
Vista last supported by 4.6

Reply:
WebRequest cant connect too Cloudflare some ssl problems 4.7 just fix

Reply:
So i test it i cant get it lower then 4.6 it can maybe run in 4.0 with third party WebHandler